docs/intro_tutorial.rst

```diff
@@ -322,16 +301,15 @@ Taking the partial trace over the second subsystem of :math:`X` yields the follo
 By default, the partial trace function in :code:`toqito` takes the trace of the second
 subsystem.
 
-.. code-block:: python
+.. jupyter-execute::
+
+     from toqito.channels import partial_trace
+     import numpy as np
+     test_input_mat = np.array(
```

Comment:
As we have done below, instead of explicitly defining the values of the matrix, we could just do:

```py
test_input_mat = np.arange(1, 17).reshape(4, 4)
```